docs/user/examples.rst

```diff
@@ -5,7 +5,7 @@ Example projects
 * Want to bootstrap a new documentation project?
 * Want to showcase your own solution?
 
-The following example projects show a rich variety of uses of Read the Docs. You can use them for inspiration, for learning and for recipies to start your own documentation projects.
+The following example projects show a rich variety of uses of Read the Docs. You can use them for inspiration, for learning and for recipies to start your own documentation projects. View the *rendered* version of each project and then head over to the Git source to see how it's done and reuse the code.
```

Comment:
Just a reminder that most of our docs use semantic line breaks: https://sembr.org/

Reply:
Ah yeah, I haven't quite worked it into my muscle memory, hopefully it will get there, soon! Thanks!